terraform/variables.tf

```diff
@@ -77,15 +93,37 @@ locals {
   app_secrets       = yamldecode(data.azurerm_key_vault_secret.app_secrets.value)
   infra_secrets     = yamldecode(data.azurerm_key_vault_secret.infra_secrets.value)
 
-  app_env_values  = try(yamldecode(file("${path.module}/workspace-variables/${var.paas_app_environment}_app_env.yml")), {})
+  app_env_values_from_yaml = try(yamldecode(file("${path.module}/workspace-variables/${var.paas_app_environment}_app_env.yml")), {})
+
+  app_env_values = merge(
+    local.app_env_values_from_yaml,
+    { DB_SSLMODE = var.db_sslmode }
+  )
 
-  custom_domain = { "CUSTOM_HOSTNAME" = "apply-${local.app_name_suffix}.london.cloudapps.digital" }
+  custom_domain    = { "CUSTOM_HOSTNAME" = "apply-${local.app_name_suffix}.london.cloudapps.digital" }
   authorized_hosts = { "AUTHORISED_HOSTS" = "apply-${local.app_name_suffix}.london.cloudapps.digital" }
 
   paas_app_environment_variables = merge(
     local.custom_domain,
     local.authorized_hosts,
-    local.app_secrets, # Values in app secrets can override anything before it
+    local.app_secrets,   # Values in app secrets can override anything before it
     local.app_env_values # Utilimately app_env_values can override anything in the merged map
   )
+  cluster = {
+    cluster1 = {
+      cluster_resource_group_name = "s189d01-tsc-dv-rg"
+      cluster_name                = "s189d01-tsc-cluster1-aks"
+      dns_suffix                  = "cluster1.development.teacherservices.cloud"
+    }
+    test = {
+      cluster_resource_group_name = "s189t01-tsc-ts-rg"
+      cluster_name                = "s189t01-tsc-test-aks"
+      dns_suffix                  = "test.teacherservices.cloud"
+    }
+    production = {
+      cluster_resource_group_name = "s189p01-tsc-ps-rg"
+      cluster_name                = "s189p01-tsc-production-aks"
+      dns_suffix                  = "production.teacherservices.cloud"
```

Comment:
Not sure how this relates to real DNS domains but the real one for production is "teacherservices.cloud". We should probably align.

Reply:
Yea that's in there because I thought we were keeping things consistent and using development, test and production names for each of the clusters.